dev/devicelab/lib/framework/cocoon.dart

```diff
@@ -100,7 +100,11 @@ class Cocoon {
     resultsJson['TestFlaky'] = isTestFlaky ?? false;
     const List<String> supportedBranches = <String>['master'];
     if(supportedBranches.contains(resultsJson['CommitBranch'])) {
-      await _sendUpdateTaskRequest(resultsJson);
+      await retry(
+            () => _sendUpdateTaskRequest(resultsJson).timeout(const Duration(seconds: 30)),
```

Comment:
Are 30 secs enough in slow network conditions?

Reply:
Anecdotally, this operation should take <3 seconds (devicelab->cocoon->skia perf+datastore). If the network is slow on this HTTP request, there are likely larger issues with the recipe. Quickly retrying is preferred.